arrows/ocv/detect_features_FAST.cxx

```diff
@@ -1,5 +1,5 @@
 /*ckwg +29
- * Copyright 2016-2018 by Kitware, Inc.
+ * Copyright 2016-2018, 2020 by Kitware, Inc.
```

Comment:
Curious why you are adding , 2020 rather than replacing 2018 with 2020...

Reply:
...because there is no 2019? I've "always" done that. (It's also possibly I am passive-aggressively trying to make the banners as ugly as possible in a bid to get official consensus to drop the whole mess :slightly_smiling_face:.)